pkg/generic/thrift/base.go

```diff
@@ -37,3 +39,55 @@ type BaseResp = base.BaseResp
 func NewBaseResp() *BaseResp {
 	return base.NewBaseResp()
 }
+
+// MergeBase mainly take frkBase and only merge the frkBase.Extra with the jsonBase.Extra
```

Comment:
what does "frk" stand for?

Reply:
And maybe we can put these APIs into internal package

Reply:
I think I can be used for other internal gateway app